packages/pglite/src/pglite.ts

```diff
@@ -293,6 +326,9 @@ export class PGlite
     }
     emscriptenOpts['pg_extensions'] = extensionBundlePromises
 
+    // Await the fs bundle
+    await fsBundleBufferPromise
```

Comment:
the only `await` between this and the initalization of this promise is the `this.fs!.emscriptenOpts` call - which also receives `emscriptenOpts` which contains a reference to the `fsBundleBuffer` which might or might not have resolved (?)

I'm assuming you're familiar with when `getPreloadedPackage` is called and this is why it's safe, otherwise I would imagine that it's not too expensive to just wait for the FS bundle where we initialize the promise for simplicity?

Reply:
Yep, it's a little confusing. `getPreloadedPackage` is called when we do:

```ts
this.mod = await PostgresModFactory(emscriptenOpts)
```

The intention is too try and download the WASM, the FS bundle and extensions all in parallel. I'll add some comments.